src/Master.cpp

```diff
@@ -91,9 +91,9 @@ void Master::run() {
   while (do_run) {
     LOG(Sev::Debug, "Master poll");
     auto p = command_listener.poll();
```

Comment:
I dislike the use of single character variable names like this one. Maybe do something like this instead:
```suggestion
    auto PollResult = command_listener.poll();
```